### DISCUSSION TITLE
Setting Windows version?

### DISCUSSION THREAD

**Discussion by asmaloney** [The-Wineskin-Project/WineskinServer#393] [Q&A] [answered]
Using winecfg I have **Windows Version** set to _Windows 11_.

When I run Steam, it thinks I'm running Windows 8.1:

<img width="281" alt="Screen Shot 2023-05-04 at 19 43 52 PM" src="https://user-images.githubusercontent.com/391371/236352918-3da3d911-83ad-4135-b1ed-f4e3a8713402.png">

How do I set this properly? I've tried using both winecfg and setting it using winetricks, but nothing seems to work.

**Comment by Gcenx** [comment, marked as answer]
It’s a known issue I’ve already resolved this for the next engine bump

**Comment by asmaloney** [comment]
Excellent - thanks Dean!

**Comment by Gcenx** [comment]
@asmaloney I’d pushed WineCX22.1.1-rc2 this will allow setting Windows 10/11 correctly.
